### ISSUE TITLE
Position stream not working!

### ISSUE THREAD

**Issue by AzarouAmine** [Baseflow/flutter-geolocator#324]
I'm trying to listen to position changes using the position stream. I've checked whether the Location Manager (Android) is enabled. This is how I'm calling the position stream:

```
Geolocator().isLocationServiceEnabled().then((loc) {
      if(loc) {
        StreamSubscription<Position> positionStream = Geolocator().getPositionStream().listen((position) {
          // This part is never reached.
          RouteApiController().routes(position).then((response) async {
            await homeBloc.updateSinkWithLoading(false);
            setRoutes(position,response,state);
          });
        });
        positionStream.cancel();
      } else {
        // If location isn't enabled!
      }
    });
```

**Comment by promistrio** [comment]
The same problem. It is not working on phone(Android 8) which i use for debug, and work on another phones, but accuracy is low regardless of settings.

**Comment by mvanbeusekom** [comment]
Fixed in version 6.0.0
